### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by tropptr-torrptrop** [Keriew/augustus#548] [General]
Hi.
I could not find any info about donations. I would like to thank you not only by "thank you" (btw, thank you! :D )
Please consider sharing something like paypal if you think it it is a good idea. If not - then.. well... Thank you for this great project!

**Comment by crudelios** [comment]
Hi, thank you for your kindness!

We are not interested in donations though.  Having fun playing the game is "donation" enough :wink:
